src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -61,7 +66,10 @@
 
     interface Computed {
         dataset: string
-        uploadFiles: Dict<UploadFile>
+        uploadFiles: Dict<UploadFile>,
+        uploadFileSections: Array<Dict<UploadFile>>,
+        outputUploadFiles: Dict<UploadFile>,
+        inputUploadFiles: Dict<UploadFile>
```

Comment:
Are these hangers-on?

Reply:
Oops, yes, relics of an earlier attempt.